README.md

```diff
@@ -1,172 +1,31 @@
-[![Python package](https://github.com//actions/workflows/python-package.yml/badge.svg)](https://github.com//actions/workflows/python-package.yml)
+# Smoothed Particle Hydrodynamics (SPH) via Inductiva API
 
-[![Documentation Status](https://readthedocs.com/projects/inductiva-research-labs-/badge/?version=latest&token=a7a7344c5bc9e511ca715449f94612e4bb7c2dff4c90931778b9c85dbbf5480c)](https://inductiva-research-labs-.readthedocs-hosted.com/en/latest/?badge=latest)
+[**Inductiva API**](https://github.com/inductiva/inductiva/tree/main) empowers users to run large-scale simulations. With a simple interface, users have access to several simulators and features that help manage complex simulation workflows. 
 
-# Start a project
+Here, we explore Smoothed Particle Hydrodynamics (SPH) with two simulators available via **Inductiva API** - [SPlisHSPlasH](https://splishsplash.physics-simulation.org/) and [DualSPHysics](https://dual.sphysics.org/). 
 
-Use this template to generate a repository for your new project. By doing so you
-will quickly have access to a couple of tools we've already set up for you.
+In this repository, we make a tour of **Inductiva API**:
```

Comment:
We are not (yet?) showing how to create a scenario. Just how to use them. In fact, I think we are missing details about how that scenario was created. I think we need to be more intentional about that because we know that these scenarios are merely toy scenarios, so what we really want to show is how we build them.